conf/test_full.config

```diff
@@ -0,0 +1,18 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running full-size tests
+ * -------------------------------------------------
+ * Defines bundled input files and everything required
+ * to run a full size pipeline test. Use as follows:
+ *   nextflow run nf-core/mhcquant -profile test_full,<docker/singularity>
+ */
+
+params {
+  config_profile_name = 'Full test profile'
+  config_profile_description = 'Full test dataset to check pipeline function'
+
+ // Input data
+  fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/mhcquant/test.fasta'
```

Comment:
I think this should be used with larger test files, some public fasta file e.g.?

Reply:
the same fasta should be fine, I changed the raw input files to real size ones now in the full test profile